### ISSUE TITLE
[BUG] Documentation Issue: Incorrect Command in Getting Started Guide

### ISSUE THREAD

**Issue by Melfos-md** [crewAIInc/crewAI#1243]
### Description

**Section:**
Getting Started - Starting a new CrewAI project  - Installing Dependencies

running `crewai install` leads to the error Message: `Error: No such command 'install'`

### Steps to Reproduce

1. Follow the Getting Started guide.
2. Run `crewai install`

### Expected behavior

I guess we should change the documentation for `poetry install`

### Screenshots/Code snippets

```
crewai install

Usage: crewai [OPTIONS] COMMAND [ARGS]...
Try 'crewai --help' for help.

Error: No such command 'install'.
```

### Operating System

Other (specify in additional context)

### Python Version

3.12

### crewAI Version

0.51.1

### crewAI Tools Version

0.8.3

### Virtual Environment

Conda

### Evidence

```
crewai install

Usage: crewai [OPTIONS] COMMAND [ARGS]...
Try 'crewai --help' for help.

Error: No such command 'install'.
```

### Possible Solution

Change the documentation

### Additional context

https://docs.crewai.com/getting-started/Start-a-New-CrewAI-Project-Template-Method/#installing-dependencies

**Comment by theCyberTech** [comment]
Already addressed in #1212

Current Docs will stay, as new version coming soon

**Comment by i-Mobyl** [comment]
Fyi, it has been almost a year, and the documentation still displays "make sure to run crewai install" in several areas.
